src/components/Person.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ name, age, sex, partnerName }) => {
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="person">
+      {name && (
+        <h2 className="person__name">
+          {`My name is ${name}`}
+        </h2>
+      )}
```

Comment:
Name is required so this check is redundant
```suggestion
     <h2 className="person__name">
        {`My name is ${name}`}
     </h2>
```